Packs/EmailCommunication/ReleaseNotes/2_0_21.md

```diff
@@ -0,0 +1,5 @@
+
+#### Scripts
+##### DisplayEmailHtmlThread
+- Updated the Docker image to: *demisto/btfl-soup:1.0.1.84651*.
+- Removed the color attribute from the HTML body.
```

Comment:
Lets change the rn to be less specific and better describe the overall change (changed the way we parse emails...)